packages/elements-core/src/components/TryIt/TryIt.tsx

```diff
@@ -203,10 +203,14 @@ export const TryIt: React.FC<TryItProps> = ({
         const contentType = response.headers.get('Content-Type');
         const type = contentType ? getResponseType(contentType) : undefined;
 
+        const bodyText = type !== 'image' ? await response.text() : undefined;
+        const blob = type === 'image' ? await response.blob() : undefined;
+
+        setResponse(undefined); // setting undefined to handle rendering large responses
         setResponse({
           status: response.status,
-          bodyText: type !== 'image' ? await response.text() : undefined,
-          blob: type === 'image' ? await response.blob() : undefined,
+          bodyText,
+          blob,
```

Comment:
Can you explain to me why this helps? 

I can kind of see how moving the awaits up, and  blocking the state update until finished could help. The `setResponse(undefined)` not so much. From my understanding React should just "batch" the 2 setResponse calls together and just render once.